### DISCUSSION TITLE
How to get VulkanMod to work under NixOS with AMD

### DISCUSSION THREAD

**Discussion by nonetrix** [xCollateral/VulkanMod#346] [Q&A] [answered]
It seems that Prismlauncher doesn't have access to Vulkan libraries because it assumes the game will use OpenGL, therefore it will fail to open. Has anyone found a workaround?

**Comment by lv37** [comment]
It works fine for me on Prismlauncher

**Reply by nonetrix** [reply]
Are you using NVIDIA?

**Reply by lv37** [reply]
No, just an Intel igpu

**Comment by sqrvrt** [comment]
Hmmm... Let's see. What drivers for your gpu do you have installed?

if it's proprietary then idk. Arch wiki suggests nvidia-utils for vulkan support which is probably proprietary. Other than that you either need nouveau (which usually supports ancient hardware) or NVK (which is still pretty unstable, even though vulkan 1.0 compatible or whatever).

So basically post logs at least

**Comment by nonetrix** [comment, marked as answer]
Fixed via https://github.com/NixOS/nixpkgs/pull/303880#event-12460580665
